src/main/java/com/spotify/docker/BuildMojo.java

```diff
@@ -493,6 +533,36 @@ private void createDockerFile(final String directory, final List<String> filesTo
     if (maintainer != null) {
       commands.add("MAINTAINER " + maintainer);
     }
+
+    if (env != null) {
+      final List<String> sortedKeys = Ordering.natural().sortedCopy(env.keySet());
+      for (String key : sortedKeys) {
+        final String value = env.get(key);
+        commands.add(String.format("ENV %s %s", key, value));
+      }
+    }
+
+    if (workdir != null) {
+      commands.add("WORKDIR " + workdir);
+    }
+
+    for (String file : filesToAdd) {
+      commands.add(String.format("ADD %s %s", file, file));
+    }
+
+    if (runList != null && !runList.isEmpty()) {
+      commands.add("RUN " + Joiner.on(" &&\\\n\t").join(runList));
```

Comment:
I'm thinking if, as a user, you wanted to have something like multiple apt operations in a single `RUN`, then you would just do `<run>apt-get update && apt-get install ...</run>`.